lib/shift4/request_options.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Shift4
+  class RequestOptions
+    class << self
+      attr_reader :idempotency_key
+    end
+
+    def initialize(params = {})
```

Comment:
Can it be typed? Ideally it will be done in the same manner as constructor for Configuration object